packages/bruno-app/src/components/RequestTabs/index.js

```
@@ -86,6 +87,17 @@ const RequestTabs = () => {
     return null;
   }
 
+  const activeTab = find(tabs, (t) => t.uid === activeTabUid);
```

Biome (2.3.13)

[error] Shouldn't redeclare 'activeTab'. Consider to delete it or rename it.

'activeTab' is defined here:


(lint/suspicious/noRedeclare)